docs/iris/src/whatsnew/2.1.rst

```diff
@@ -53,23 +53,24 @@ Iris 2.1 Features
 * Added :meth:`iris.analysis.trajectory.interpolate` that allows you to
   interpolate to find values along a trajectory.
 * It is now possible to add an attribute of ``missing_value`` to a cube.
-  https://github.com/SciTools/iris/issues/1588
+  (:issue:`1588`)
 * Iris can now represent data on the Albers Equal Area Projection,
-  and the NetCDF loader and saver were updated to handle this.
-  https://github.com/SciTools/iris/issues/2943
+  and the NetCDF loader and saver were updated to handle this. (:issue:`2943`)
 * The :class:`~iris.coord_systems.Mercator` projection has been updated to accept
-  the ``standard_parallel`` keyword argument.
+  the ``standard_parallel`` keyword argument. (:pr:`3041`)
 
 Bugs Fixed
 ==========
 
 * All var names being written to NetCDF are now CF compliant.
   Non alpha-numeric characters are replaced with '_', and var names now always
-  have a leading letter. https://github.com/SciTools/iris/pull/2930
+  have a leading letter. (:pull:`2930`)
```

Comment:
It's a stylistic thing only but I'd prefer to see the brackets containing the link _inside_ the sentence; something like:

```
* :mod:`iris.quickplot` labels now honour the axes being drawn to when using the ``axes`` keyword (:pull:`3010`).
```

(I've chosen a shorter example from below for the sake of brevity.)